demos/Test_hydra.py

```
@@ -0,0 +1,80 @@
+import pandas as pd 
+import hydra
+from omegaconf import DictConfig, OmegaConf
+
+OmegaConf.register_new_resolver('eq', lambda x, y: x.lower()==y.lower())
+OmegaConf.register_new_resolver('contains', lambda x, y: x.lower() in y.lower())
+OmegaConf.register_new_resolver('resolve_default', lambda default, arg: default if arg=='' else arg)
+
+
+@hydra.main(version_base=None, config_path="../configs", config_name="config")
+def parse_hydra_config(cfg: DictConfig):
+    for k in cfg.paths.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)